src/components/RandomDog.js

```diff
@@ -1,20 +1,47 @@
-import React from 'react';
-import './RandomDog.css';
+import React from "react";
+import "./RandomDog.css";
 
 class RandomDog extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      randomDog: [],
```

Comment:
Since `randomDog` is always a string, you should set it to be a string here:

```
randomDog: '',
```

Reply:
Thank you.. I have changed it.. :)